packages/viewer/src/lib/interactions/interactionApiOrthographic.ts

```diff
@@ -164,62 +170,77 @@ export class InteractionApiOrthographic extends InteractionApi<OrthographicCamer
     point: Point.Point,
     delta: number
   ): Promise<void> {
-    return this.transformCamera(({ camera, viewport, frame, depthBuffer }) => {
-      if (
-        this.orthographicZoomData == null ||
-        Point.distance(point, this.orthographicZoomData.startingScreenPt) > 2
-      ) {
-        const frameCam = camera.toFrameCamera();
-        const dir = frameCam.direction;
-        const ray = viewport.transformPointToOrthographicRay(
-          point,
-          frame.image,
-          frameCam
-        );
+    return this.transformCamera(
+      ({ camera, viewport, frame, depthBuffer, boundingBox }) => {
+        if (
+          this.orthographicZoomData == null ||
+          Point.distance(point, this.orthographicZoomData.startingScreenPt) > 2
+        ) {
+          const frameCam = camera.toFrameCamera();
+          const dir = frameCam.direction;
+          const ray = viewport.transformPointToOrthographicRay(
+            point,
+            frame.image,
+            frameCam
+          );
```

Comment:
It's strange that we have `transformPointToOrthographicRay` and `transformPointToRay`. You could pass an incorrect camera type to either of these. Instead, I'd suggest just having a single `transformPointToRay` and that method inspects what type of camera is passed to it.

Also not sure why we have an interface for `FrameCameraWithMatrices`. It seems awkward to have this interface in addition to `FrameCameraBase`.

Reply:
Also, understand this isn't from your changes. I'll create a ticket to follow up.

Reply:
I agree. I created https://vertexvis.atlassian.net/browse/PLAT-4344.